generated/nidaqmx/system/watchdog.py

```diff
@@ -94,7 +92,7 @@ def expiration_states(self):
         return self._expiration_states
 
     @property
-    def expir_trig_dig_edge_edge(self):
+    def dig_edge_watchdog_expir_trig_edge(self):
```

Comment:
normal task trigger name would be something like this `triggers.arm_start_trigger.dig_edge_edge`, so lets go with `expiration_trigger_dig_edge_edge`